### PR TITLE
Breakpoints Option String Tweaks

### PR DESCRIPTION
**Pull request by jarcelo**
Part of issue #2000

### CODE REVIEW

**Comment by jarcelo** [comment]
> To address the DCO requirement you'll need to sign-off the commit(s):
> 
> * https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin
> * https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

Thanks. Just added commit signature.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 1]
```diff
@@ -931,17 +931,17 @@ brk.dialogue.confirmDropMessage.button.confirm.label = Drop
 brk.dialogue.confirmDropMessage.button.cancel.label = Cancel
 brk.name = Breakpoint Extension
 brk.optionspanel.name             = Breakpoints
-brk.optionspanel.option.alwaysOnTop.label	= ZAP always on top when breakpoint hit
+brk.optionspanel.option.alwaysOnTop.label	= ZAP Always on Top When Breakpoint Hit
```
Comment (kingthorin) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:934]: I'd suggest leaving this one and adding a period.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 2]
```diff
@@ -931,17 +931,17 @@ brk.dialogue.confirmDropMessage.button.confirm.label = Drop
...
-brk.optionspanel.option.confirmDropMessage.label = Confirm drop trapped message
-brk.optionspanel.option.inScopeOnly.label = Only break on messages in scope
-brk.optionspanel.option.showBreakFilteringButtons.label = Show buttons to select the requests you don't want ZAP to break on
-brk.optionspanel.option.javaScriptUrlRegex.label = Javascript URL regex:
-brk.optionspanel.option.cssAndFontsUrlRegex.label = CSS and Fonts URL regex:
-brk.optionspanel.option.multimediaUrlRegex.label = Multimedia URL regex:
-brk.optionspanel.option.notpossibletoshowtip = Can not show this buttons when break buttons are only in the toolbar
+brk.optionspanel.option.breakmode.dual.label	= Separate Request and Response Buttons
+brk.optionspanel.option.breakmode.simple.label	= Single Combined Button
+brk.optionspanel.option.confirmDropMessage.label = Confirm Drop Trapped Message
```
Comment (kingthorin) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:938]: I'd suggest leaving this one and adding a period.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 3]
```diff
@@ -931,17 +931,17 @@ brk.dialogue.confirmDropMessage.button.confirm.label = Drop
...
+brk.optionspanel.option.inScopeOnly.label = Only Break on Messages in Scope
```
Comment (kingthorin) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:939]: I'd also suggest leaving this one and adding a period.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 4]
```diff
@@ -931,17 +931,17 @@ brk.dialogue.confirmDropMessage.button.confirm.label = Drop
...
+brk.optionspanel.option.showBreakFilteringButtons.label = Show Buttons to Select the Requests You Don't Want ZAP to Break On
```
Comment (kingthorin) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:940]: I'd suggest this one should be a sentence.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 5]
```diff
@@ -931,17 +931,17 @@ brk.dialogue.confirmDropMessage.button.confirm.label = Drop
...
+brk.optionspanel.option.javaScriptUrlRegex.label = Javascript URL Regex:
+brk.optionspanel.option.cssAndFontsUrlRegex.label = CSS and Fonts URL Regex:
+brk.optionspanel.option.multimediaUrlRegex.label = Multimedia URL Regex:
+brk.optionspanel.option.notpossibletoshowtip = Buttons Are Disabled When Break Buttons Are Only in the Toolbar
```
Comment (kingthorin) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:944]: This one should also just be a sentence.

**Review by kingthorin — CHANGES_REQUESTED** [review]
Sorry it took me so long to get through this.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties (hunk 1 above)
Comment (jarcelo) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties]: Thanks. Updated commit and implemented suggestion.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties (hunk 2 above)
Comment (jarcelo) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties]: Thanks. Updated commit and implemented suggestion.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties (hunk 3 above)
Comment (jarcelo) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties]: Thanks. Updated commit and implemented suggestion.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties (hunk 4 above)
Comment (jarcelo) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties]: Thanks. Updated commit and implemented suggestion.

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties (hunk 5 above)
Comment (jarcelo) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties]: Thanks. Updated commit and implemented suggestion.

**Review by kingthorin — APPROVED** [review]
Thanks

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 6]
```diff
@@ -931,17 +931,17 @@ brk.dialogue.confirmDropMessage.button.confirm.label = Drop
...
-brk.optionspanel.option.notpossibletoshowtip = Can not show this buttons when break buttons are only in the toolbar
+brk.optionspanel.option.breakmode.dual.label	= Separate Request and Response Buttons
+brk.optionspanel.option.breakmode.simple.label	= Single Combined Button
+brk.optionspanel.option.confirmDropMessage.label = Confirm drop trapped message.
+brk.optionspanel.option.inScopeOnly.label = Only break on messages in scope.
+brk.optionspanel.option.showBreakFilteringButtons.label = Show buttons to select the requests you don't want ZAP to break on.
+brk.optionspanel.option.javaScriptUrlRegex.label = Javascript URL Regex:
+brk.optionspanel.option.cssAndFontsUrlRegex.label = CSS and Fonts URL Regex:
+brk.optionspanel.option.multimediaUrlRegex.label = Multimedia URL Regex:
+brk.optionspanel.option.notpossibletoshowtip = Buttons are disabled when break buttons are only in the toolbar.
```
Comment (thc202) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:944]: Why is the meaning being changed?

Comment (kingthorin) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:944]: Hmmm I didn't think it changed the meaning significantly, where/when does this show?

Comment (thc202) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:944]: The filter buttons are only shown in the Break tab, if the break buttons are only shown in the main toolbar they (filter) will not be visible.

Comment (kingthorin) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:944]: Does this seem better?

```suggestion
brk.optionspanel.option.notpossibletoshowtip = Cannot show these buttons when break buttons are only shown in the main toolbar.
```

Comment (thc202) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:944]: Yes.

Comment (kingthorin) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:944]: @jarcelo could you tweak that?

**Review by kingthorin — APPROVED** [review]

**Comment by jarcelo** [comment]
Thanks @thc202 and @kingthorin for your review and PR change suggestions! Appreciate it.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Pull request merged**